app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DownvotesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @vote = current_user.votes.new
+    @review = Review.find(params[:review_id])
+    sitcom = @review.sitcom
+    @vote.review_id = params[:review_id]
+
+    unless current_user.votes.where(review_id: @review.id).length > 1
```

Comment:
generally I don't see it as good practice to use an `unless` on more than a one liner.